dev/sync-uv

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# rely on `uv` in path
+if ! command -v uv &> /dev/null; then
+    echo "Installing uv ..."
+    pip install uv
```

Comment:
I suggest we lock the version of uv in the installation script. For example:

```bash
pip install uv==0.6.12
```

What do you think?

Reply:
I would prefer not to pin the exact version of uv tool, as uv releases versions frequently.
As we do in `dev/poetry` in  Poetry time, here is providing minium help to continue the execution.
Prevent over-tuned in specific scope, especially on the developer's client side.

Meanwhile, we do pindown the version of uv in CI tests to gurantee all the commands are good to use.